yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -643,11 +643,19 @@ func (d *IndexIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 						"",
 					))
 				} else if isHotspotType && idx == 0 {
+					//If first column is hotspot type then only report hotspot issue
 					hotspotIssues, err := reportHotspotsOnIndexes(param.ExprCastTypeName, obj.GetObjectType(), obj.GetObjectName())
 					if err != nil {
 						return nil, err
 					}
 					issues = append(issues, hotspotIssues...)
+				} else if isHotspotType && !slices.Contains(queryparser.RangeShardingClauses, param.SortByOrder) {
```

Comment:
maybe for now it's okay (in the interest of time), but we seem to be combining the concerns of "which is a type which can lead to hotspot problem" and "which is a type for which we should nudge user to use range indexes". 

Currently, both of these are the same. But they might not be. 
pls add a todo.

Reply:
Yeah right, will add a TODO